Code/Tools/Standalone/Source/LUA/LUAEditorContext.cpp

```diff
@@ -2468,4 +2479,67 @@ namespace LUAEditor
     {
         return AzFramework::StringFunc::Path::IsExtension(assetPath.c_str(), ".lua");
     }
+
+    void Context::DeleteBreakpointAll(const AZStd::string& fromAssetId)
+    {
+        BreakpointMap::iterator iter;
+        bool isSend = false;
+        for (iter = (m_pBreakpointSavedState->m_Breakpoints).begin(); iter != (m_pBreakpointSavedState->m_Breakpoints).end(); iter++)
+        {
+            if ((static_cast<Breakpoint>(iter->second)).m_assetId == fromAssetId)
+            {
+                DeleteBreakpoint(iter->first);
+                isSend = true;
+            }
+        }
+
+        if (!isSend)
+        {
+            EBUS_EVENT(LUAEditorDebuggerMessages::Bus, RemoveBreakpoint, "", 0);
+        }
+    }
+
+    void Context::DeleteBreakpointSingle(const AZStd::string& fromAssetId, int lineNumber)
```

Comment:
You can probably combine this function with ```DeleteBreakpointAll``` and instead delete all breakpoints if the lineNumber is -1 or something (even default it to -1)